packaging/src/docker/6.x/nginx.conf

```diff
@@ -0,0 +1,15 @@
+server {
```

Comment:
Comment on why and what is the purpose of the nginx

Reply:
nginx used here as a load balancer for single or multiple slaves for all quires from Alfresco. This is optional when we deploy along with slave config.

Reply:
Please add the comment in the code.